src/main/java/core/basesyntax/dao/Storage.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.dao;
+
+import java.util.HashMap;
+
+public class Storage {
+    public static final HashMap<String, Integer> storage = new HashMap<>();
```

Comment:
Follow the encapsulation principle. It's a bad practice to make your class fields public or package-private. Consider changing the access modifier to private and provide public methods to interact with the storage.